content/en/infrastructure/livecontainers.md

```diff
@@ -13,73 +13,100 @@ further_reading:
 
-### Kubernetes Resources
-
-[Kubernetes Resources for Live Containers][1] is currently in private beta. Fill out [this form][5] to request access.
-
-If you're using Kubernetes, enable Kubernetes Resources for Live Containers to gain multi-dimensional visibility into all Kubernetes workloads across your clusters. Inspired by the `kubectl` tool, this feature gives you complete coverage of your Kubernetes infrastructure in a continuously updated table with curated resource metrics, faceted search, per-workload detailed view, and visualized maps.
-
-## Installation
-
-Follow the [Docker][6] or [Kubernetes][7] Agent installation instructions. Enable the Process Agent to populate your Live Containers view. Container metrics are available without additional configuration after installation.
-
-**Kubernetes Resources for Live Containers requires installation of**:
-
-* [Datadog Agent][8] version 7.21.1 (or above)
-* [Datadog Cluster Agent][9] 1.9.0 (or above)
+## Configuration
 
 ### Kubernetes Resources
 
-To enable Kubernetes Resources for Live Containers, follow the [Helm instructions][10] and add the following changes to your `values.yaml` file:
-
-{{< code-block lang="yaml" filename="values.yaml" >}}
-datadog:
-  ...
-  processAgent:
-    enabled: true
-  ...
-  orchestratorExplorer:
-    enabled: true
-...
-clusterAgent:
-  enabled: true
-  image:
-    repository: datadog/cluster-agent
-    tag: latest
-    pullPolicy: Always
-...
-agents:
-  image:
-    repository: datadog/agent
-    tag: latest
-    pullPolicy: Always
-...
-{{< /code-block >}}
-
-In cases where the Agent is not able to automatically detect the Kubernetes cluster name, you must set it in `values.yaml`:
-
-{{< code-block lang="yaml" filename="values.yaml" >}}
-datadog:
-   ...
-   clusterName: <PLACEHOLDER>
-   ...
-{{< /code-block >}}
-
-**Note**: The cluster name must be 80-characters or less.
-
-On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the Agent and the cluster Agent don't have access to the cloud metadata APIs, or the cluster name is longer than 80 characters.
+The Agent and [Cluster Agent][5] can be configured to retrieve Kubernetes resources for [Live Containers][6]. This feature allows you to monitor the state of pods, deployments and other Kubernetes concepts in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+
+Kubernetes resources for Live Containers requires [Agent version >= 7.21.1][7] prior to the configurations below.
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+If you are using the official [Datadog Helm Chart][1]:
+
+- Use chart version 2.4.5 or above
+- Set `datadog.orchestratorExplorer.enabled` to true in [values.yaml][2]
+- Deploy a new release
+
+
+[1]: https://github.com/DataDog/helm-charts
+[2]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "DaemonSet" %}}
```

Comment:
not sure we should call this DaemonSet, because it requires the agent daemonset and the cluster agent deployment. If we want to distinguish this method from the Helm install I'd go with "raw manifests" or something similar

Reply:
@cohenyair Thoughts on this? I think DaemonSet makes sense for tabbing based off of the initial beta docs / the Jira card request for use to flesh out into Helm, DaemonSet and Operator tabs going forward.

Reply:
I agree with @hkaj that using DaemonSet can be misleading, but since we're using it in many places in the docs and the product, we should stick to it in the short term here as well (for consistency)